src/main/resources/openapi/EMSIntegrationAPIs.yaml

```diff
@@ -879,42 +879,7 @@ components:
           type: array
           description: The list of reasons the ERO rejected the application.
           items:
-            $ref: '#/components/schemas/RejectedReasonItem'
-    RejectedReasonItem:
-      title: Rejected Reason Item
-      description: |
-        Contains electors friendly text and ERO's prefefence if the reason should be included in comms.
-      type: object
-      properties:
-        electorReason:
-          description: Elector friendly reason text. This text is curated and managed within EROP.
-          type: string
-          maxLength: 100
-        type:
-          type: string
-          description: | 
-            A static value that can be used by EMSs to add additional business logic if needed.
-            The current values are below but may change overtime, hence this is not an enforced restricted list / enum.
-            Should EROP add a new 'type' or revise one, we would not want EMSs to require a deployment.
-            As such these values should not be used for business logic in processing applications, but could be used for aggregations of rejection reasons across applications.
-            * identity-not-confirmed
-            * signature-is-not-acceptable
-            * fraudulent-application (will be passed with includeInElectorComms:false)
-            * not-registered-to-vote
-            * not-eligible-for-reserved-polls
-            * dob-not-provided
-            * incomplete-application
-            * proxy-not-registered-to-vote
-            * proxy-limits
-            * other (will be passed with includeInElectorComms:false)
-          maxLength: 50
-        includeInComms:
-          type: boolean
-          description: |
-            Indicates whether this reason should be included in comms to the Elector and this field is sufficient alone - EMS implementations do not need to consider the type as well.
-            EROP will automatically ensure this is false for any reason that should never be included in comms.
-            EROP may provide EROs flexibility to select which reasons they would like be sent to the elector.
-          default: true
+            $ref: './sqs/Models/RejectedReasonItem.yaml'
```

Comment:
We shouldn't change this even if is the same, EMS providers use this file directly so we shouldn't do imports particularly from internal models which we want the freedom to change